Makefile

```diff
@@ -1042,6 +1045,21 @@ geojson/albers/city-labels.geojson:
 				name:name \
 		> $@
 
+tiles/wapo-2016-election-centroids.mbtiles: geojson/albers/centroid-states.geojson \
+	geojson/albers/centroid-counties.geojson \
+	geojson/albers/centroid-districts.geojson
+	mkdir -p $(dir $@)
+	tippecanoe --projection EPSG:3857 \
+		-f \
+		--named-layer=states:geojson/albers/centroid-states.geojson \
```

Comment:
Gonna go with `states-centroids`, `counties-centroids`, `districts-centroids` since the other layers are plural too.